docs/reference/indices/rollover-index.asciidoc

```diff
@@ -73,10 +81,10 @@ index creation applies to the rollover action.
 [[rollover-index-api-path-params]]
 ==== {api-path-parms-title}
 
-`<alias>`::
+`<rollover-target>`::
 (Required, string)
```

Comment:
For parameters that are required in some situations, we typically add an asterisk.
```suggestion
(Required*, string)
```